.github/ISSUE_TEMPLATE/graduation.md

```diff
@@ -0,0 +1,387 @@
+Sub-Projects: kyverno-json, chainsaw, policy-reporter, reports-server, kyverno-envoy-plugin, playground
+Communication: #kyverno, #kyverno-dev on Kubernetes Slack, #kyverno on CNCF slack, [community meetings](https://kyverno.io/community/#community-meetings), mailing list (cncf-kyverno-maintainers@lists.cncf.io)
+
+Project points of contacts:
+* Shuting Zhao, shuting@nirmata.com
+* Jim Bugwadia, jim@nirmata.com
+* Kyverno maintainers mailing list: cncf-kyverno-maintainers@lists.cncf.io
+* #kyverno-dev channel on Kubernetes Slack
+
+## Graduation Criteria Summary for Kyverno
+
+### Adoption Assertion
+
+_The project has been adopted by the following organizations in a testing and integration or production capacity:_
+
+### Criteria
+
+## Application Process Principles
+
+### Suggested
+
+N/A
+
+### Required
+
+- [ ] **Give a presentation and engage with the domain specific TAG(s) to increase awareness**
+    - [ ] TAG Security
+    - [ ] TAG Contributor Strategy
+
+<!-- (Project assertion goes here) --> 
+
+- [ ]  **TAG provides insight/recommendation of the project in the context of the landscape**
+
+<!-- (Project assertion goes here) --> 
+
+- [x]  **All project metadata and resources are [vendor-neutral](https://contribute.cncf.io/maintainers/community/vendor-neutrality/).**
+
+<!-- (Project assertion goes here) --> 
+
+- [x] **Review and acknowledgement of expectations for graduated projects and requirements for moving forward through the CNCF Maturity levels.**		
+   - [x] Met during Project's application on 08-12-2023, initial Graduation PR [Proposal: Kyverno to Graduation #1217](https://github.com/cncf/toc/pull/1217)
+
+<!-- (Project assertion goes here) --> 
+
+Completion of this due diligence document, resolution of concerns raised, and presented for public comment satisifies the Due Diligence Review criteria.
+
+- [x] **Additional documentation as appropriate for project type, e.g.: installation documentation, end user documentation, reference implementation and/or code samples.**
+
+<!-- (Project assertion goes here) --> 
+
+## Governance and Maintainers
+
+Note: this section may be augmented by the completion of a Governance Review from TAG Contributor Strategy.
+
+### Suggested
+
+- [x]  **Governance has continuously been iterated upon by the project as a result of their experience applying it, with the governance history demonstrating evolution of maturity alongside the project's maturity evolution.**
+
+<!-- (Project assertion goes here) --> 
+
+The strategies for major Project Governance were discussed and refined during the Kyverno maintainers' meeting. You can find the meeting notes [here](https://docs.google.com/document/d/1I_GWsz32gLw8sQyuu_Wv0-WQrtRLjn9FuX2KGNkvUY4/edit):
+* 05-21-2024
+* 02-20-2024
+* 12-05-2023
+
+The Governance page has been maintained and updated through Kyverno website's Github repo, history:
+
+https://github.com/kyverno/website/commits/main/content/en/community/_index.md
```

Comment:
```suggestion
https://kyverno.io/community/
```

Reply:
Pointed to the `kyverno/community` repo.